frontend/src/hooks/useMaxMultiplier.js

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
The package is not listed in the dependencies, which is causing build issues and failing CI. Could you please fix it?